SPECS/k3s/install.patch

```diff
@@ -1,57 +0,0 @@
-From bd240e73ab1f815dcd2aac4b1bcd5d022c581aa9 Mon Sep 17 00:00:00 2001
```

Comment:
Why did we remove this patch ?
The Title of the patch says it might be necessary for building `VHDX`.
Did we verify the `VHDX` build ?

Let's do this:
- Verify if the `VHDX` builds & works without this patch
- Refresh & apply this patch & then verify sanity of `VHDX` via a build + test

Reply:
I removed the patch as the changes in it were already included in vendor_build.patch. I have verified and tested all my changes without this patch itself.

Reply:
Thanks for testing the changes :)